lib/kernel/test/rpc_SUITE.erl

```diff
@@ -48,7 +48,7 @@
 %% This module needs to compile on old nodes...
 -define(CT_PEER(), {ok, undefined, undefined}).
 -define(CT_PEER(Opts), {ok, undefined, undefined}).
--define(CT_PEER(Opts, Release, PrivDir), {ok, undefined, undefined}).
+-define(CT_PEER_REL(Opts, Release, PrivDir), {ok, undefined, undefined}).
```

Comment:
While at it, maybe also introduce ?CT_PEER/4 that accepts optional arguments map, that can have some future extensibility (e.g. "releases root").

Also, while I intentionally left `?CT_PEER/3` undocumented, it may happen that it's quite useful for other users (at least I used that in my own compatibility testing, having a dozen of OTP versions installed into default location). Maybe it's worth considering documenting it and making a public API someday.